tools/dev-benchmark-pallet.sh

```diff
@@ -79,6 +79,13 @@ TEMPLATE_PATH=benchmarking/frame-weight-runtime-template.hbs \
     tools/benchmarking.sh "$PALLET" "*" --check
 cp -v tmp/flashbox_weights/$PALLET.rs runtime/flashbox/src/weights/$PALLET.rs
 
+# Starlight weights
+BINARY=target/release/tanssi-relay \
+    TEMPLATE_PATH=benchmarking/frame-weight-runtime-template.hbs \
+    CHAIN=starlight-dev \
+    OUTPUT_PATH=tmp/starlight_weights \
+    tools/benchmarking.sh "$PALLET" "*" --check
```

Comment:
This script is for quickly updating weights after adding a new extrinsic to a pallet. It currently won't work for Starlight because we are not using custom weights yet. Therefore, you can ignore it in this PR.

Reply:
Sure thing removed it in https://github.com/moondance-labs/tanssi/pull/675/commits/c8c1a7e05c8448e578ff180720f11c04a1f813f5